scripts/extract_dts_includes.py

```diff
@@ -404,6 +404,20 @@ def extract_single(node_address, yaml, prop, key, prefix, defs, def_label):
 
   return
 
+def extract_string_prop(yaml, node_address, key, label, defs):
+
+  prop_def = {}
+  node = reduced[node_address]
+  prop = node['props'][key]
+
+  k = convert_string_to_label(key).upper()
+  prop_def[label] = "\"" + prop + "\""
+
+  if node_address in defs:
+    defs[node_address].update(prop_def)
+  else:
+    defs[node_address] = prop_def
+
```

Comment:
hmm, need to look at that.  What exactly is extract_property trying to do.  The stringification seems to happen ok in extract_single, but what's going on with the label from the yaml I don't quite understand.